### DISCUSSION TITLE
Radial linecut Pole-Figures

### DISCUSSION THREAD

**Discussion by mardilos** [mtex-toolbox/mtex#1854] [Ask Anything]
Hi all,

is it possible to do a radial linecut in a pole-figure, to extract the orientation spread of the measurement points?

In our samples, we have a 110 orientation in RD. I want to measure the width of these poles to get an idea how far some grains are tilted in in-plane direction
![Pole-Figure_Au](https://github.com/mtex-toolbox/mtex/assets/65020763/583c7a97-bbf0-4e18-8a3e-4013219e5953)

Best
mardilos

**Comment by kilir** [comment]
Hi,
this sounds a bit like https://github.com/mtex-toolbox/mtex/discussions/806
despite this does not extract an orientation spread which is not possible from a pole figure but the values along a line.

I remember there were some other discussions&solutions on that topic here was well, so searching the discussion list may help.

Hope this helps,
Rüdiger
